### PR TITLE
Fix ScrollPanel render calls and usage

### PR DESCRIPTION
**Pull request by iTitus**
The scrollbar was rendered with a texture shader and scrolling didn't work.
I fixed both issues.

### CODE REVIEW

**Comment by iTitus** [comment]
~There is still an issue with the "Done" button not being clickable when not fully scrolled down.~

EDIT: fixed

**Comment by iTitus** [comment]
Should be working now

**Review by marchermans — APPROVED** [review]
Good catch with the mod list scrollbar.

Thanks for the fix.

File: src/main/java/net/minecraftforge/client/gui/ScrollPanel.java [hunk 1]
```diff
@@ -227,21 +228,21 @@ public void render(PoseStack matrix, int mouseX, int mouseY, float partialTicks)
...
             tess.end();
-            worldr.begin(VertexFormat.Mode.QUADS, DefaultVertexFormat.POSITION_TEX_COLOR);
+            worldr.begin(VertexFormat.Mode.QUADS, DefaultVertexFormat.POSITION_COLOR);
             worldr.vertex(barLeft,            barTop + barHeight, 0.0D).uv(0.0F, 1.0F).color(0x80, 0x80, 0x80, 0xFF).endVertex();
             worldr.vertex(barLeft + barWidth, barTop + barHeight, 0.0D).uv(1.0F, 1.0F).color(0x80, 0x80, 0x80, 0xFF).endVertex();
             worldr.vertex(barLeft + barWidth, barTop,             0.0D).uv(1.0F, 0.0F).color(0x80, 0x80, 0x80, 0xFF).endVertex();
             worldr.vertex(barLeft,            barTop,             0.0D).uv(0.0F, 0.0F).color(0x80, 0x80, 0x80, 0xFF).endVertex();
             tess.end();
-            worldr.begin(VertexFormat.Mode.QUADS, DefaultVertexFormat.POSITION_TEX_COLOR);
+            worldr.begin(VertexFormat.Mode.QUADS, DefaultVertexFormat.POSITION_COLOR);
```
Comment (gigaherz) [inline src/main/java/net/minecraftforge/client/gui/ScrollPanel.java:245]: You changed these to POS_COLOR but the vertices are still calling .uv()

**Review by gigaherz — CHANGES_REQUESTED** [review]
Minor issue but for consistency.

**Review by gigaherz — APPROVED** [review]

**Pull request merged**